From 2a3a27ca12f6a75a122d7220521b189645d0dbf9 Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Wed, 23 Apr 2025 08:46:24 +0200
Subject: [PATCH 826/841] fix(stm32mp2-fdts): remove eMMC support on
 stm32mp215f-dk.

Remove SDMMC2 node for stm32mp215f-dk board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 2c8a03067a83e3cb018b052a5987f2ed31986e18 fix(stm32mp2-fdts): remove eMMC support on stm32mp215f-dk.]
Change-Id: Ic795c829827dc653a39e117630681af1dfcaba8f
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/453476
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
---
 fdts/stm32mp215f-dk.dts | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/fdts/stm32mp215f-dk.dts b/fdts/stm32mp215f-dk.dts
index 5e587bb50..963d41b04 100644
--- a/fdts/stm32mp215f-dk.dts
+++ b/fdts/stm32mp215f-dk.dts
@@ -187,19 +187,6 @@
 	status = "okay";
 };
 
-&sdmmc2 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&sdmmc2_b4_pins_a &sdmmc2_d47_pins_a>;
-	non-removable;
-	no-sd;
-	no-sdio;
-	st,neg-edge;
-	bus-width = <8>;
-	vmmc-supply = <&v3v3>;
-	vqmmc-supply = <&vddio2>;
-	status = "okay";
-};
-
 &usart2 {
 	pinctrl-names = "default", "idle", "sleep";
 	pinctrl-0 = <&usart2_pins_a>;
-- 
2.47.2

